crates/fig_os_shim/src/fs.rs

```diff
@@ -478,10 +636,66 @@ mod tests {
                 assert_eq!(append($a, $b), PathBuf::from($expected));
             };
         }
-        assert_append!("/abc/test", "/test", "/abc/test/test");
-        assert_append!("/tmp/.dir", "/tmp/.dir/home/myuser", "/tmp/.dir/home/myuser");
-        assert_append!("/tmp/.dir", "/tmp/hello", "/tmp/.dir/tmp/hello");
-        assert_append!("/tmp/.dir", "/tmp/.dir/tmp/.dir/home/user", "/tmp/.dir/home/user");
+        #[cfg(unix)]
+        {
+            assert_append!("/abc/test", "/test", "/abc/test/test");
+            assert_append!("/tmp/.dir", "/tmp/.dir/home/myuser", "/tmp/.dir/home/myuser");
+            assert_append!("/tmp/.dir", "/tmp/hello", "/tmp/.dir/tmp/hello");
+            assert_append!("/tmp/.dir", "/tmp/.dir/tmp/.dir/home/user", "/tmp/.dir/home/user");
+        }
+
+        #[cfg(windows)]
+        {
+            // Basic path joining
+            assert_append!("C:\\abc\\test", "test", "C:\\abc\\test\\test");
```

Comment:
I'm not sure of the actual intent of joining two absolute paths. Is this still relevant for Windows?